drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -141,6 +141,7 @@ class Hmis::Hud::Project < Hmis::Hud::Base
     )
   end
 
+  # Legacy referrals
   scope :receiving_referrals, -> do
```

Comment:
this is only used in a couple places, perhaps rename to `receiving_legacy_referrals` or `accepting_legacy_referrals`? (I see the existing code uses language "receive" but the new code uses "accept" to mean the same thing. I think "accept" is fine it would be good to be consistent going forward)

Reply:
Hm, do you have a preference between the two? Now that I'm thinking about it, I slightly prefer `receiving` over `accepting` because it's more neutral regarding the separate concept of "referral acceptance," later on down the road.

Meaning, these scopes `receiving_referrals` and flags `receives_referrals?` aren't saying anything about whether a potential direct referral will get accepted, only that the project can receive it and acceptance may or may not happen later.